addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -866,7 +898,7 @@ public void run() {
                     // the CCU1 does not support the ping command, we need a workaround
                     getRpcClient(getDefaultInterface()).listBidcosInterfaces(getDefaultInterface());
                     // if there is no exception, connection is valid
-                    pongReceived();
+                    connectionConfirmed();
```

Comment:
pongReceived() is required, with connectionConfirmed() the CCU1 connection tracker restarts every 15sec and reloads all metadata.

Reply:
Oh, I see now. This fix was relatively new and I probably did not have that locally yet. So something must have gone wrong during rebasing.